src/components/Label/TopicPageLabel.js

```diff
@@ -21,20 +21,21 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     marginStart: 20,
     alignSelf: 'flex-start',
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     height: normalize(29)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the background color and the addition of `color: COLORS.white2` for the text color in the `domainText` style seem to be intended to improve contrast and readability. However, please ensure that these color changes are consistent with the overall design language and accessibility guidelines of your application.

```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: COLORS.almostBlack,
     height: normalize(29)
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->